src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2431,10 +2431,6 @@ MavlinkReceiver::handle_message_hil_state_quaternion(mavlink_message_t *msg)
 		matrix::Quatf q(hil_state.attitude_quaternion);
 		q.copyTo(hil_attitude.q);
 
-		hil_attitude.rollspeed = hil_state.rollspeed;
-		hil_attitude.pitchspeed = hil_state.pitchspeed;
-		hil_attitude.yawspeed = hil_state.yawspeed;
```

Comment:
you should probably publish onto `vehicle_angular_velocity` here

Reply:
Yes I suppose so.

Reply:
I'll do gyro below along with accel.

Reply:
Done.